app/assets/stylesheets/styles/components/pages/_featured_crops.css.scss

```diff
@@ -0,0 +1,42 @@
+.featured-crops {
+  background-color: rgb(240, 245, 240);
+  padding: 40px inherit 20px;
+
+  .featured-crops-grid {
+    margin-top: 30px;
+
+    .featured-crop-container {
+      margin-bottom: 30px;
+
+      .featured-crop {
```

Comment:
This means that things might be a bit too specific, which might be an indicator that you can refactor here without losing anything, and ultimately make things more modular.